docs/src/security.md

```diff
@@ -297,13 +297,13 @@ section of the Kubernetes documentation for further information.
 CloudNativePG exposes ports at operator, instance manager and operand
 levels, as listed in the table below:
 
-System           | Port number  | Exposing            |  Name               |  Certificates  |  Authentication
-:--------------- | :----------- | :------------------ | :------------------ | :------------  | :--------------
-operator         | 9443         | webhook server      | `webhook-server`    |  TLS           | Yes
-operator         | 8080         | metrics             | `metrics`           |  no TLS        | No
-instance manager | 9187         | metrics             | `metrics`           |  no TLS        | No
-instance manager | 8000         | status              | `status`            |  no TLS        | No
-operand          | 5432         | PostgreSQL instance | `postgresql`        |  optional TLS  | Yes
+System           | Port number  | Exposing            |  Name               |  Certificates                  |  Authentication
+:--------------- | :----------- | :------------------ | :------------------ | :----------------------------  | :--------------
+operator         | 9443         | webhook server      | `webhook-server`    |  TLS                           | Yes
+operator         | 8080         | metrics             | `metrics`           |  no TLS                        | No
+instance manager | 9187         | metrics             | `metrics`           |  no TLS                        | No
+instance manager | 8000         | status              | `status`            |  TLS (no TLS in old releases)  | Yes
```

Comment:
Authentication is still no. We verify the server part, but the client side is not authenticated.

Reply:
🤦